### DISCUSSION TITLE
ChatGLM3-6B回答中英文混杂，ChatGLM-6B都没有出现过类似的情况，是哪里出问题了？

### DISCUSSION THREAD

**Discussion by PoisonousBromineChan** [zai-org/ChatGLM3#635] [Q&A] [answered]
<img width="1725" alt="abb3bf910c21114b3cfd606fe3071a2" src="https://github.com/THUDM/ChatGLM3/assets/64314088/2b7d8ce5-e650-456a-a931-8f2f090c894a">

**Comment by catmeowjiao** [comment]
正常，等新版本

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
等下一个 版本吧，这个是模型问题
